tests/fute/Makefile

```diff
@@ -0,0 +1,33 @@
+PYTHON_CMD:=source venv/bin/activate && source .secrets && python3
+
+all: venv check
+
+venv:
+	python3 -m venv venv
+	source venv/bin/activate && pip3 install https://github.com/CSCfi/tauhka/archive/master.zip
+	source venv/bin/activate && pip3 install -r requirements.txt
+
+chromedriver:
+	curl -O https://chromedriver.storage.googleapis.com/74.0.3729.6/chromedriver_mac64.zip
```

Comment:
I wonder if there's a good way to automatically install the correct version and keep it up to date with the installed Chrome version. Would it be possible to have a copy of a specific version of Chrome for testing?

Reply:
hmm.. there was something for that hmm..

Reply:
yeah there was some misc python projects, but all what they did is that they have the same url and some fixed value for the version. I rather see that this is a separate topic to be solved. at the moment there is 75 out there and it works with web driver for 74.

Reply:
it was noted in private chat, that there is API change between 75 and 74 WebDriver that will cause the tests to fail. To have support for 75 series an update to Tauhka is required.